src/RoleObjects/MenuItemData.js

```diff
@@ -35,7 +36,8 @@ export default class MenuItemData extends AccessibilityObject {
 
   /**
    * Informs a menu label of which DisplayObject is the submenu
-   * @access package
+   * @access public
```

Comment:
please see previous comments about package level being the correct access level

Reply:
@package similar to    @access package
http://usejsdoc.org/tags-package.html#examples